docs/tutorials/microbit-basic.md

```diff
@@ -0,0 +1,107 @@
+1. Get a pen and paper, press the A/B buttons and write down the numbers
+1. Follow our [Getting started with the Telenor NB-IoT Developer Platform guide](getting-started.html) and use the IMEI/IMSI from the step above
+1. Leave the browser tab with live device data open, so we can see data we send in the next section
+
+## Install nbiot extension
+1. Go to [makecode.microbit.org](https://makecode.microbit.org/)
+1. Click on the big *«New Project»-button*
+1. Click the *gear* in the top right, then *Extensions*
+    - ![Extensions screenshot](img/makecode-extensions.png)
+1. In the search field, enter the URL below and click the search button
+    1. https://github.com/ExploratoryEngineering/pxt-nbiot
+    1. Select «nbiot» in the search result
+    - ![nbiot extension](img/makecode-nbiot-install.png)
+1. You should now get an «NB-IoT» category in the blocks menu
+    - ![nbiot menu](img/makecode-nbiot-menu.png)
+
+
+## Hello World
+Drag-n-drop blocks from the menu to reproduce the blocks below:
+
+![Hello World example](img/microbit-hello.png)
+
+### What's going on here?
+In the block editor, every chain of blocks starts with an event (something happened). All the blocks inside of it will be executed in order from top to bottom.
+
+“on start” is only run once, when the micro:bit starts. So here you typically add blocks that should be run before any other code, and only needs to run once. For the NB-IoT extension to work, we need to tell it what pins on the micro:bit it is connected to, and what server IP and port to send data to.
+
+“on nbiot connected” is run once, when the device is successfully connected to the NB-IoT nettwork, and get an IP address from our NB-IoT Development Platform. If the device is not registered (see steps above), you'll not get an IP address and will not be able to connect. First show a heart in the LED display to indicate that we connected, then send the text string «Hello World!». If you still have the browser tab from the NB-IoT Developer Platform open (see [Register Device](#register-device)), you should see that some data has arrived. If it shows up as «SGVsbG8gV29ybGQh», you need to tick the «Decode base 64 payload» checkbox to show it as text.
+
+![Hello world in NB-IoT Developer Platform](img/payload-hello.png)
+
+## Sending useful data
+Sending a hard-coded text isn't that useful other than to check that everything works as it should. The fun really begins when we start connecting the physical world with the digital. So try to send some sensor data (like temperature), or send a message when something happens (like a button press or that the micro:bit moves). The micro:bit has a few sensors built in, but if you feel adventorous you can also add other sensors by connecting to one of the pins (definitely get an edge connector breakout board for that). Remember that NB-IoT is meant for small data amounts, so try to keep the size of the data small and don't send too frequently.
```

Comment:
Nit picking really, but isn't it spelled adventurous? Or am I sparking the whole color vs colour debate?